main.go

```diff
@@ -160,12 +160,20 @@ func SlightEchoHandler(writer http.ResponseWriter, request *http.Request) {
 	writer.Write(responsePayload)
 }
 
-func runWorker(c *cli.Context, n *server.Node) error {
+func runWorker(c *cli.Context, n *server.Node, g peer.ID) error { // TODO: Is there a reason all these have just c,n as arguments?
 	time.Sleep(10 * time.Second)
+	var f, release = n.Cluster.View().Lookup(c.Context) // TODO: Submit a query for ("gatewayID/lb/chan")?
+	defer release()
+	var record, err = f.Await(c.Context) // Confused because record is wrong type, so how get chan capability from record?
```

Comment:
For the avoidance of doubt:  you're getting the chan capability from the call to `n.Vat.Connect`.  You can't get it from a `routing.Record`.